nextflow.config

```diff
@@ -34,6 +34,12 @@ params {
     sequence_filter             = null
     save_bam                    = false
 
+    // CheckM options
+    skip_binqc                  = true
+    checkm_download_url         = "https://data.ace.uq.edu.au/public/CheckM_databases/checkm_data_2015_01_16.tar.gz"
```

Comment:
Perhaps a bit brittle, but this hasn't been updated for quite a while, right?

Reply:
I followed MAG pipeline on this : https://github.com/nf-core/mag/blob/2.5.4/nextflow.config see line 129